YoCode/FeatureDetailsStorage.cs

```diff
@@ -42,7 +42,7 @@ public FeatureDetailsStorage(TestType mode)
                     new FeatureDetails
                     {
                         FeatureTitle = "Bad input crashes have been fixed",
-                        FeatureWeighting = 1
+                        FeatureWeighting = 0
```

Comment:
Numeric types are always initialised to `0` in .Net so I don't think you need all these zeroes.